packages/framework/build.xml

```diff
@@ -104,6 +105,24 @@
         </copy>
     </target>
 
+    <target name="check-annotations" description="Checks whether annotations of extended classes in the project are ok. Reported problems can be fixed using 'fix-annotations' phing target">
```

Comment:
According to our naming conventions for phing targets, this should be named `annotations-check` (and same for fix down below)

https://github.com/shopsys/shopsys/blob/rv-mg-website-docs/docs/contributing/guidelines-for-phing-targets.md#naming-conventions

Reply:
renamed in a new commit (I do not want to experience the living hell during the rebase and renaming it through history :grin: ) d1fb0382d

Reply:
Nice!